setup.cfg

```
@@ -29,9 +29,9 @@ keywords =
 [options]
 packages = find:
 install_requires =
-    Wagtail >=2,<3
+    Wagtail >=2.15,<3
     django-storages[boto3] <2
-python_requires = >=3.5
+python_requires = >=3.6
```

```suggestion
python_requires = >=3.7
```

https://endoflife.date/python - Python 3.6 has EOLed 4.5 months ago, so may as well tidy this too

---

Wagtail LTS is compatible with 3.6. I don't know if there's a benefit to be gained from being too restrictive here.

---

works for me